SonarQube.TeamBuild.PostProcessor/AssemblyFriendInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleToAttribute("SonarQube.TeamBuild.PostProcessor.Tests")]
```

Comment:
Using InternalsVisibleTo caused problems with the signing process for //Build so we stopped using it.